typings/index.d.ts

```diff
@@ -1203,6 +1261,23 @@ declare module 'discord.js' {
     public toJSON(): object;
   }
 
+  export class MessageButton extends BaseMessageComponent {
+    constructor(data?: MessageButton | MessageButtonOptions);
+    public customID: string | null;
+    public disabled: boolean;
+    public emoji: unknown | null;
+    public label: string | null;
+    public style: MessageButtonStyle | null;
+    public type: 'BUTTON';
+    public url: string | null;
+    public setCustomID(customID: string): this;
+    public setDisabled(disabled: boolean): this;
+    public setEmoji(emoji: EmojiIdentifierResolvable): this;
+    public setLabel(label: string): this;
+    public setStyle(style: MessageButtonStyle | number): this;
```

Comment:
Since we have the enum:
```suggestion
    public setStyle(style: MessageButtonStyle | MessageButtonStyles): this;
```